locale/en/blog/release/v0.10.0.md

```diff
@@ -136,7 +136,7 @@ The benchmarks below were run on an Apple 13-inch, Late 2011 MacBook
 Pro with a 2.8 GHz Intel Core i7 processor, 8GB of 1333MHz DDR3 RAM,
 running OS X Lion 10.7.5 (11G63b).  The numbers are slightly different
 on Linux and SmartOS, but the conclusions are the same.  The [raw data
-is available](http://nodejs.org/benchmarks-v0.10-vs-v0.8/), as well.
+is available](https://nodejs.org/benchmarks-v0.10-vs-v0.8/), as well.
```

Comment:
404. This must have come form a different source, as it wasn't part of the old nodejs.org website repo.